code/game/objects/machinery/vending/new_marine_vendors.dm

```diff
@@ -710,6 +713,97 @@
 
 
 
+/obj/machinery/marine_selector/clothes/synth
+	name = "M57 Synthetic Equipment Vendor"
+	desc = "An automated synthetic equipment vendor hooked up to a modest storage unit."
+	icon_state = "marineuniform"
+	vendor_role = "Synthetic"
+	lock_flags = JOB_LOCK
+
+	listed_products = list(
+							list("STANDARD EQUIPMENT", 0, null, null, null),
+							list("Essential Synthetic Set", 0, /obj/effect/essentials_set/synth, MARINE_CAN_BUY_ESSENTIALS, "white"),
+							list("UNIFORM (choose 1)", 0, null, null, null),
+							list("TGMC marine uniform", 0, /obj/item/clothing/under/marine, MARINE_CAN_BUY_UNIFORM, "black"),
```

Comment:
```
			if(bitf == MARINE_CAN_BUY_UNIFORM)
				new /obj/item/radio/headset/almayer/marine(loc, H.assigned_squad.name, vendor_role)
				new /obj/item/clothing/gloves/marine(loc, H.assigned_squad.name, vendor_role)
				//if(istype(SSticker.mode, /datum/game_mode/ice_colony))//drop a coif with the uniform on ice colony
				if(SSmapping.configs[GROUND_MAP].map_name == MAP_ICE_COLONY)
					new /obj/item/clothing/mask/rebreather/scarf(loc)
```
This will break, since synths do not have an assigned squad.